anmol098/waka-readme-stats
# Allow selection for another branch than the default ones

Fixes #302

# How to test
1. Add PUSH_BRANCH_NAME: 'branch name' to the action configuration or the run environment
2. Let it run
3. The action should commit to the branch specified
